pinot-core/src/main/java/org/apache/pinot/core/data/manager/realtime/RealtimeSegmentDataManager.java

```diff
@@ -275,7 +275,7 @@ public void deleteSegmentFile() {
   private final int _partitionGroupId;
   private final PartitionGroupConsumptionStatus _partitionGroupConsumptionStatus;
   final String _clientId;
-  private final RecordEnricherPipeline _recordEnricherPipeline;
+  private final RecordTransformer _recordEnricherPipeline;
```

Comment:
Same here, we should be able to integrate it into the `TransformPipeline`

Reply:
Got it. I will take a look into TransformPipeline. Didn't know this class existed. Will modify according to it.